.gitignore

```diff
@@ -38,3 +38,6 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+
+#Ignore db test files
+db/test.*
```

Comment:
What are these files?  I see they are binary files

Reply:
These are auto generated files that got created while setting up the tests  see screenshot attached.
<img width="1440" alt="Screenshot 2021-04-07 at 17 10 28" src="https://user-images.githubusercontent.com/81353313/113890295-50358100-97c4-11eb-8e1c-d4fad5d5b654.png">